docs/content/docs/ops/traces.md

```diff
@@ -97,7 +97,7 @@ Flink reports a single span trace for the whole checkpoint once checkpoint reach
   </thead>
   <tbody>
     <tr>
-      <th rowspan="6">org.apache.flink.runtime.checkpoint.CheckpointStatsTracker</th>
+      <th rowspan="14">org.apache.flink.runtime.checkpoint.CheckpointStatsTracker</th>
```

Comment:
NIT: this field is so wide that I don't see the Description column; I can scroll horizontally, but would only do that if I know there is a column on the right.

Don't know how to fix that easily though 🤷‍♂️

Reply:
use an abbreviation `o.a.f`? 🤔 Could we use a tooltip to expand the scope? For example display `(...).CheckpointStatsTracker` that expands in the tooltip to the fully qualified name?

Reply:
Good idea, should work!

Reply:
I've added line breaks to the scope. More readable and the effect is the same.